src/GraphQL/Resolver/CompleteUploadResolver.php

```
@@ -0,0 +1,28 @@
+
+namespace App\GraphQL\Resolver;
+
+use GraphQL\Type\Definition\ResolveInfo;
+
+final class CompleteUploadResolver
+{
+    /**
+     * @param array<string,mixed> $args
+     * @return array<string,mixed>
+     */
+    public function resolve(mixed $rootValue, array $args, mixed $context, ResolveInfo $info): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$rootValue'. (undefined)

(UnusedFormalParameter)